src/codelabs/dart-cheatsheet.md

```diff
@@ -1667,22 +1653,32 @@ void main() {
   }
   
   if (!caughtException) {
-    errs.add('Called FirstTwoLetters(\'\') and didn\'t get an exception from the failed assertion.');
+    errs.add('Called FirstTwoLetters(\'\') and didn\'t get an exception \n from the failed assertion.');
   }
   
   if (errs.isEmpty) {
-    _result(true);
+    print('Success!');
   } else {
-    _result(false, errs);
+    errs.forEach(print);
   }
 }
-{$ end test.dart $}
-{$ begin hint.txt $}
-Two assignments need to happen:
-letterOne should be word[0], and letterTwo should be word[1].
-{$ end hint.txt $}
+{$ end main.dart $}
 ```
 
+<details markdown="1">
+ <summary>
+   <b>Solution for initializer lists example</b>
+  </summary>
+  Two assignments need to happen:
+  `letterOne` should be `word[0]`, and `letterTwo` should be `word[1]`.
```

Comment:
Do you think it's worth being more specific here?

```suggestion
  `letterOne` should be assigned to `word[0]`, and
  `letterTwo` should be assigned to `word[1]`.
```

Reply:
Agree with specifying the action, but wouldn't it be more correct to say it the other way around? Like, I'd read that code as "`word[0]` is assigned to `letterOne`", so I guess the instruction would be:

`letterOne` should be assigned `word[0]`

I put this in my edit but lmk if that's less accurate